sections/getting_started.tex

```diff
@@ -2,31 +2,24 @@ \section{Getting Started}
 
 \subsection{Installing Rust}
 
-\begin{frame}[c]{Rustup.rs}
-\begin{quote}
-«rustup is an installer for \\
-the systems programming language Rust» \\
-\vspace{0.5em}
-{\normalfont \small --- www.rustup.rs}
-\end{quote}
-\end{frame}
+\begin{frame}[fragile]{Rustup}
+	The best way to install and update the Rust toolchain is rustup
+	(\url{https://rustup.rs/}).
+
+	It makes it easy to:
+
+	\begin{itemize}
+		\item Install different Rust versions
+		\item Install cross compilation toolchains
+	\end{itemize}
+
+	\vspace{1em}
+
+	To download the installer script:
 
-\begin{frame}[fragile]{Rustup.rs}
-\begin{itemize}
-    \item Official installation method by now
-    \item Makes it easy to:
-      \begin{itemize}
-        \item Install different Rust versions
-        \item Install cross compilation toolchains
-      \end{itemize}
-    \item Written in Rust itself
-    \pause\item Installing is easy:
 \begin{minted}{sh}
-$ curl https://sh.rustup.rs -sSf | sh
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs
 \end{minted}
-    \pause Obviously you shouldn't do that ;)
-    \item Alternatively you can use \url{https://play.rust-lang.org/}
-\end{itemize}
```

Comment:
While I'm OK with removing the silly joke about not executing stuff directly from the internet, I'd leave the mention of https://play.rust-lang.org/.

Reply:
Ok, added it back!